kuma/wiki/migrations/0001_initial.py

```diff
@@ -285,7 +284,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='document',
             name='team',
-            field=models.ForeignKey(blank=True, to='teamwork.Team', null=True),
+            field=models.ForeignKey(blank=True, to='wiki.DocumentTag', null=True),
```

Comment:
Ah, yeah. The label is definitely ambiguous. I chose the name *not ready to be merged* in Discord for that reason.

I'll stay out of the way now. :smile: